packages/workflow/src/Interfaces.ts

```diff
@@ -1020,9 +1021,19 @@ export interface INodePropertyTypeOptions {
 	showAlpha?: boolean; // Supported by: color
 	sortable?: boolean; // Supported when "multipleValues" set to true
 	expirable?: boolean; // Supported by: hidden (only in the credentials)
+	resourceMapper?: ResourceMapperTypeOptions;
 	[key: string]: any;
 }
 
+export interface ResourceMapperTypeOptions {
+	resourceMapperMethod: string;
+	mode: 'add' | 'upsert';
```

Comment:
should we have also update mode?

Reply:
We should. Will update.

Reply:
@michael-radency updated in the latest commit